e2e-tests/insurance/cypress/e2e/journeys/your-business/alternative-trading-address/alternative-trading-address.spec.js

```diff
@@ -81,9 +88,18 @@ context('Insurance - Your business - Alternative trading address page - I want t
       cy.checkText(partials.headingCaption(), CONTENT_STRINGS.HEADING_CAPTION);
     });
 
+    it('renders a registered office address heading', () => {
+      cy.checkText(fieldSelector(COMPANY_ADDRESS).label(), fields[COMPANY_ADDRESS].LABEL);
+    });
+
+    it('renders a registered office address', () => {
+      const addressText = `${address.addressLine1}${address.addressLine2}${address.locality}${address.region}${address.postalCode}`;
+      cy.checkText(fieldSelector(COMPANY_ADDRESS).id(), addressText);
```

Comment:
I don't think we should use `fieldSelector` here because its not a form field. I think a partial would be better, what do you think?